### ISSUE TITLE
Unable to use IdRecognizer with USDL

### ISSUE THREAD

**Issue by christinachimi** [BlinkID/blinkid-flutter#50]
I am trying to update to 6.x but so far unable to. The USDL recognizer doesn't seem to work anymore (filed separate issue for that), so I thought I could use the IdBarcodeRecognizer. I'm not able to get that to work, due to this error

```
flutter: type '_Map<String, dynamic>' is not a subtype of type 'BarcodeElements?'
flutter:
#0      new IdBarcodeRecognizerResult
id_barcode_recognizer.dart:172
#1      IdBarcodeRecognizer.createResultFromNative
id_barcode_recognizer.dart:226
#2      MicroblinkScanner.scanWithCamera
microblink_scanner.dart:42
<asynchronous suspension>
```

I think you need a `BarcodeElements(nativeResult["extendedElements"])` in the `IdBarcodeRecognizerResult` constructor. 

Is there any way to get a patch change like this fast-tracked? I'm currently blocked on this update. Thanks!

**Comment by somyaguptagit** [comment]
Hi @christinachimi Are you trying to upgrade BlinkID flutter to version 6.1.2? If yes, we suggest using BlinkIDMultiSideRecognizer since it has all the new features included in it. 
Also, you can forward all the issues you are experiencing while upgrading to v6.1.2, to support@microblink.com and we would be happy to assist you further.

**Comment by christinachimi** [comment]
Ok, I can forward them along. We are trying to achieve feature parity first, and the multi side recognizer is not what we use today. that would involve UX and training changes for our users.
